EIPS/eip-3014.md

```diff
@@ -0,0 +1,48 @@
+---
+eip: 3014
+title: Create `eth_symbol` method for JSON-RPC
+author: Peter Grassberger (@PeterTheOne)
+discussions-to: https://github.com/ethereum/EIPs/issues/3012
+status: Draft
+type: Standards Track
+category: Interface
+created: 2020-09-30
+---
+
+## Simple Summary
+Add `eth_symbol` method to the JSON-RPC that returns the symbol of the native coin of the network.
+
+## Abstract
+The new method `eth_symbol` (`eth_`-namespaced) has no parameters and returns a string of the native coin of the network. For the Ethereum mainnet this will be `ETH`, other networks will have other symbols.
+
+## Motivation
+Wallets that deal with multiple networks need some basic information for every blockchain that they connect to. One of those things is the symbol of the native coin of the network. Instead of requiring the user to research and manually add the symbol it could be provided to the wallet via this proposed JSON-RPC endpoint and used automatically. There are lists of networks with symbols like https://github.com/ethereum-lists/chains where a user can manually look up the correct values. But this information could easily come from the network itself.
+
+## Specification
+Method: `eth_symbol`.
+
+Params: none.
+
+Returns: `result` - the native coin symbol, string
+
+Example:
+
+```js
+curl -X POST --data '{"jsonrpc":"2.0","method":"eth_symbol","params":[],"id":1}'
+
+// Result
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": "ETH"
+}
+```
+
+## Rationale
+This endpoint is similar to [EIP-695](./eip-695) but it provides the symbol instead of `chainId`. It provides functionality that is already there for [ERC-20](./eip-20) tokens, but not yet for the native coin of the network. Alternative naming of `eth_nativeCurrencySymbol` was considered, but the context and the fact that it just returns one value makes it clear that that it returns the symbol for the native coin of the network.
```

Comment:
```suggestion
This endpoint is similar to [EIP-695](./eip-695.md) but it provides the symbol instead of `chainId`. It provides functionality that is already there for [ERC-20](./eip-20.md) tokens, but not yet for the native coin of the network. Alternative naming of `eth_nativeCurrencySymbol` was considered, but the context and the fact that it just returns one value makes it clear that that it returns the symbol for the native coin of the network.
```